great_expectations/datasource/fluent/data_asset/data_connector/data_connector.py

```diff
@@ -40,6 +40,10 @@ class DataConnector(ABC):
         data_asset_name: The name of the DataAsset using this DataConnector instance
     """
 
+    # needed to select the asset level kwargs needed to build the DataConnector
+    asset_level_option_keys: ClassVar[tuple[str, ...]] = ()
+    asset_options_type: ClassVar[Type] = dict
```

Comment:
@Kilo59 If possible, could you please provide a rich docstring for `asset_options_type` and `asset_level_option_keys` here in the top-level `DataConnector` module?  The reason for this request is that whenever a reader sees a class variable, which does not appear to be used anywhere in the present module, the experience of interpreting the code slows down appreciably.  We have an analogous situation in the Metrics component (I have made some effort at documenting the class variables used, but a lot more is in order).  On a separate thought, is the class variable the best way of achieving our goal here?  Could this be an indicator that a better / cleaner design is lurking somewhere and we just have not yet nailed it?  Thank you.